form/src/main/kotlin/com/ritense/form/service/PrefillFormService.kt

```diff
@@ -307,17 +307,17 @@ class PrefillFormService(
                     JsonPointer.valueOf("/$propertyName")
                 if (container.contains("/{indexOf")) {
                     val indexValueJsonPointer = getIndexValueJsonPointer(container)
-                    val id = placeholders.at(indexValueJsonPointer).textValue()
+                    val id = placeholders!!.at(indexValueJsonPointer).textValue()
```

Comment:
What's the point of making `placeholders` and `source` nullable when it's going to crash the code when you use `null`?

Reply:
if `placeholders` and `source` are null, this code must be unreachable otherwise it will throw an error. This happens when a form doesn't need prefilling.

Bu I have changed it.